docs/framework/performance/lazy-initialization.md

```diff
@@ -78,35 +78,35 @@ manager: "wpickett"
   
 > [!NOTE]
->  If you create a <xref:System.Lazy%601> object with the `isThreadSafe` constructor parameter set to `false` or the `mode` constructor parameter set to <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName>, you must access the <xref:System.Lazy%601> object from a single thread or provide your own synchronization. This applies to all aspects of the object, including exception caching.  
+>  If you create a <xref:System.Lazy%601> object with the `isThreadSafe` constructor parameter set to `false` or the `mode` constructor parameter set to <xref:System.Threading.LazyThreadSafetyMode.None?displayProperty=fullName>, you must access the <xref:System.Lazy%601> object from a single thread or provide your own synchronization. This applies to all aspects of the object, including exception caching.  
   
- As noted in the previous section, <xref:System.Lazy%601> objects created by specifying <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> treat exceptions differently. With <xref:System.Threading.LazyThreadSafetyMode>, multiple threads can compete to initialize the <xref:System.Lazy%601> instance. In this case, exceptions are not cached, and attempts to access the <xref:System.Lazy%601.Value%2A> property can continue until initialization is successful.  
+ As noted in the previous section, <xref:System.Lazy%601> objects created by specifying <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> treat exceptions differently. With <xref:System.Threading.LazyThreadSafetyMode.PublicationOnly>, multiple threads can compete to initialize the <xref:System.Lazy%601> instance. In this case, exceptions are not cached, and attempts to access the <xref:System.Lazy%601.Value%2A> property can continue until initialization is successful.  
   
  The following table summarizes the way the <xref:System.Lazy%601> constructors control exception caching.  
   
 |Constructor|Thread safety mode|Uses initialization method|Exceptions are cached|  
 |-----------------|------------------------|--------------------------------|---------------------------|  
-|Lazy(T)()|(<xref:System.Threading.LazyThreadSafetyMode>)|No|No|  
-|Lazy(T)(Func(T))|(<xref:System.Threading.LazyThreadSafetyMode>)|Yes|Yes|  
-|Lazy(T)(Boolean)|`True` (<xref:System.Threading.LazyThreadSafetyMode>) or `false` (<xref:System.Threading.LazyThreadSafetyMode>)|No|No|  
-|Lazy(T)(Func(T), Boolean)|`True` (<xref:System.Threading.LazyThreadSafetyMode>) or `false` (<xref:System.Threading.LazyThreadSafetyMode>)|Yes|Yes|  
+|Lazy(T)()|(<xref:System.Threading.LazyThreadSafetyMode.None>)|No|No|  
```

Comment:
This should be ExecutionAndPublication, not None.

Reply:
Thanks @rpetrusha, I'll commit these changes now